i3_agenda/__init__.py

```diff
@@ -79,10 +83,11 @@ def main():
 
     closest = get_closest(events)
     if closest is None:
-        print("No events")
```

Comment:
why did you remove it?

Reply:
Oups. It was not intended to be part of the PR. It is just a personal preference (My bar is quite full and I prefer to have no message than a "No events" indicator).
I can add another switch for this in another PR.